exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,7 @@
-// Write your function here
-
+function createGreeting(name) {
+  return greet + name;
+}
+var greet = "Hello, my name is ";
```

Comment:
It's better that this variable declaration moves inside the `createGreeting` function.
Also `const` is preferable to `var` in this case as a variable declared with `const` can't be overwritten.